health/notifications/email/README.md

```diff
@@ -30,4 +30,26 @@ sudo su -s /bin/bash netdata
 
 Where `[ROLE]` is the role you want to test. The default (if you don't give a `[ROLE]`) is `sysadmin`.
 
+# Simple SMTP transport configuration
+
+If you want an alternative of sendmail in order to have a simple MTA configuration for sending emails and auth to an existing SMTP server :
+
+- Remove all sendmail stuff.
```

Comment:
Do you mean this?
Uninstall `sendmail`

Reply:
Yes because I don't use it, msmtp act as a simple MTA and I map the binary of smtp to sendmail for softwares using it like yours